carbon-design-system/carbon-for-ibm-dotcom
# feat(rtl-arrow): Fix arrows used for CTA's when in RTL

### Related Ticket(s)

Closes #11725 
[Jira Ticket](https://jsw.ibm.com/browse/ADCMS-4922)

### Description

Checkes `document.dir` and uses the appropriate arrow icon for the text direction (right in LTR and left in RTL). For many stories involving CTA's or CTA derivatives, avoid using the slotted icon when all we're going for is `cta-type="local"`, in other words the arrow icon.

### Testing instructions

1. Navigate to any Story making use of a CTA and/or button
2. Compare LTR and RTL
3. In LTR, where the arrow icon is used, the arrow should be pointing right
4. In RTL, where the arrow icon is used, the arrow should be pointing left

### Changelog

**Changed**

- Change the direction of the CTA arrow when in RTL languages.
